Makefile

```
@@ -1,4 +1,4 @@
-.PHONY: help deps dev test test-live-smoke lint format build build-binary build-bundle prepare-release-files install clean
+.PHONY: help test lint format build build-bundle install clean
```

checkmake (0.2.2)

[warning] Missing required phony target "all"

(minphony)